src/integrationTest/resources/ccd/response/sample-case.json

```diff
@@ -1,5 +1,5 @@
 {
-  "id": 111,
+  "id": 1539007368674134,
```

Comment:
This json represents a found case. As we're attaching supplementary evidence to a case by this case ref, it has to be what the tests expect (previously it was the envelope's case ref that mattered).